cli/ops/dns.rs

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2020.
+ * Relta, LLC. - All rights reserved.
+ */
```

Comment:
```suggestion
// Copyright 2020 the Deno authors. All rights reserved. MIT license.
```

Reply:
Yes sorry about the copyrights, they will be updated before I'm ready to merge -- my IDE just does it automatically.